styles/drop-down-list/_styles.scss

```diff
@@ -0,0 +1,60 @@
+//== drop-down-list
+//
+//## styles for drop-down-list
+
+.daptiv-drop-down-list {
+    display: inline-block;
+    font: $dropdownlist-font;
+    line-height: $dropdownlist-item-height;
+    max-width: $dropdownlist-item-max-width;
+    min-width: $dropdownlist-item-min-width;
+    position: relative;
+
+    .content {
+        border: solid $dropdownlist-border-size $dropdownlist-border-color;
+        border-top: 0;
+        position: absolute;
+        top: $dropdownlist-item-height + 2px;
+        width: 100%;
+    }
+
+    .item {
+        background: $color-base-0;
+        color: $color-text-dark;
+        display: block;
+        margin: $dropdownlist-item-border-size;
+        overflow: hidden;
+        padding: 0 $dropdownlist-item-padding;
+        text-decoration: none;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        width: 100%;
+
+        &:not(.current):hover {
```

Comment:
:nit: would probably be more efficient to make this just a child selector of the .content class